src/helpers/utils.ts

```diff
@@ -48,3 +48,11 @@ export async function getEmailAddresses(email: string): Promise<{ address: strin
 export async function getUniqueEmails(): Promise<{ email: string }[]> {
   return await db.queryAsync('SELECT DISTINCT email FROM subscribers WHERE verified > 0');
 }
+
+export function isValidEmail(input: string) {
+  return new RegExp(/^[A-Za-z0-9_!#$%&'*+\/=?`{|}~^.-]+@[A-Za-z0-9.-]+$/, 'gm').test(input);
+}
+
+export function isValidAddress(input: string) {
```

Comment:
can we replace it with https://docs.ethers.org/v5/api/utils/address/#utils-isAddress ?

Reply:
Pushed on #49 by mistake

Reply:
Pushed also in this PR